watertap/examples/flowsheets/RO_with_energy_recovery/multi_sweep.py

```diff
@@ -0,0 +1,111 @@
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+from watertap.tools.parameter_sweep import PredeterminedFixedSample, parameter_sweep
+import watertap.examples.flowsheets.RO_with_energy_recovery.RO_with_energy_recovery as RO
+from watertap.examples.flowsheets.RO_with_energy_recovery.RO_with_energy_recovery import (
+    ERDtype,
+)
+
+
+def set_up_sensitivity():
+    outputs = {}
+
+    m = RO.build(erd_type=ERDtype.pump_as_turbine)
+    RO.set_operating_conditions(m)
+    RO.initialize_system(m)
+    RO.solve(m)
+    m.fs.feed.properties[0].flow_mass_phase_comp.unfix()
+    m.fs.feed.properties[0].flow_vol_phase["Liq"].fix()
+    m.fs.feed.properties[0].conc_mass_phase_comp["Liq", "NaCl"].fix()
+    RO.optimize_set_up(m)
+    m.fs.eq_minimum_water_flux.deactivate()
+    RO.solve(m)
+    RO.display_system(m)
+    RO.display_design(m)
+
+    # uncomment to create outputs
+
+    outputs["LCOW"] = m.fs.costing.LCOW
+    # outputs["RO Water Flux"] = m.fs.RO.flux_mass_phase_comp[0, 1, "Liq", "H2O"]
+    # outputs["RO Membrane Area"] = m.fs.RO.area
+    # outputs["RO Energy Consumption"] = m.fs.costing.specific_energy_consumption
+    # outputs["System Capital Cost"] = m.fs.costing.aggregate_capital_cost
+    # outputs["RO Capital Cost"] = m.fs.RO.costing.capital_cost
+    # outputs["Pump Capital Cost"] = m.fs.P1.costing.capital_cost
+    # outputs["ERD Capital Cost"] = m.fs.ERD.costing.capital_cost
+    # outputs["RO Operating Cost"] = m.fs.RO.costing.fixed_operating_cost
+    # outputs[
+    #     "MLC Operating Cost"
+    # ] = m.fs.costing.maintenance_labor_chemical_operating_cost
+    # outputs["Feed Flow Rate"] = m.fs.feed.properties[0].flow_vol_phase["Liq"]
+    # outputs["Permeate Flow Rate"] = m.fs.product.properties[0].flow_vol_phase["Liq"]
+    # outputs["Retentate Flow Rate"] = m.fs.disposal.properties[0].flow_vol_phase["Liq"]
+    # outputs["RO Operating Pressure"] = m.fs.RO.inlet.pressure[0]
+    # outputs["RO Permeate H2O Mass Flow"] = m.fs.RO.permeate.flow_mass_phase_comp[
+    #     0, "Liq", "H2O"
+    # ]
+    # outputs["RO Permeate Salt Mass Flow"] = m.fs.RO.permeate.flow_mass_phase_comp[
+    #     0, "Liq", "NaCl"
+    # ]
+    # outputs["RO Retentate H2O Mass Flow"] = m.fs.RO.retentate.flow_mass_phase_comp[
+    #     0, "Liq", "H2O"
+    # ]
+    # outputs["RO Retentate Salt Mass Flow"] = m.fs.RO.retentate.flow_mass_phase_comp[
+    #     0, "Liq", "NaCl"
+    # ]
+
```

Comment:
Do we want to keep these commented outputs? I know it's used for different simulations, but might make more sense in a local version.

Reply:
I'd like to keep these here just so that I _don't_ have to keep a local version (and neither does anyone else who wants to run the TEA analysis we did for the dye desal team). But if we think it really should be removed, I can do that.